build-aux/toolchain/config.sh

```diff
@@ -0,0 +1,36 @@
+
+#Set default values
+if [[ -z "$ARCH" ]]; then
+    ARCH="i386"
+fi
+
+if [[ -z "$TARGET" ]]; then
+    TARGET="$ARCH-elf"
+fi
+
+if [[ -z "$COMPILER" ]]; then
+    COMPILER="gcc"
+fi
+
+# Build directory
+TOOLCHAIN_DIRECTORY=toolchain
+INSTALL_DIRECTORY=$(pwd)/$TOOLCHAIN_DIRECTORY
```

Comment:
Unclear what's the difference between `TOOLCHAIN_DIRECTORY` and `INSTALL_DIRECTORY`, either one of those must be configurable or it should be one unique variable IMO.

Reply:
done